Oye/Fade/Fade/sketch_jul12a/sketch_jul12a.ino

```diff
@@ -0,0 +1,25 @@
+int led = 11;          // the PWM pin the LED is attached to
+int brightness = 0;    // how bright the LED is
+int fadeAmount = 5;    // how many points to fade the LED by
```

Comment:
Its really easy to understand because your notes are super neat! I agree with James (I also need to do the same) and add the same detailed notes to your Blink. You got this Oye!